src/highrl/envs/eval_env.py

```diff
@@ -3,98 +3,181 @@
 from argparse import Namespace
 from configparser import RawConfigParser
 
+from highrl.utils.abstract import Position
 from highrl.envs.robot_env import RobotEnv
 from highrl.obstacle.single_obstacle import SingleObstacle
 
 
 class RobotEvalEnv(RobotEnv):
+    """Evaluation environment for robot performance"""
+
     def __init__(self, config: RawConfigParser, args: Namespace):
         super(RobotEvalEnv, self).__init__(config, args)
```

Comment:
Rewrite this to conform with the new PEP8 style rules:
```python
super().__init__(config, args)
```

Reply:
Done